forge-ai/src/main/java/forge/ai/ability/DamageDealAi.java

```diff
@@ -893,7 +892,11 @@ private boolean damageChooseRequiredTargets(final Player ai, final SpellAbility
 
             // See if there's an indestructible target that can be used
             CardCollection indestructible = CardLists.filter(ai.getCardsIn(ZoneType.Battlefield),
-                    Predicates.and(CardPredicates.Presets.CREATURES, CardPredicates.Presets.PLANESWALKERS, CardPredicates.hasKeyword(Keyword.INDESTRUCTIBLE), CardPredicates.isTargetableBy(sa)));
+                    CardPredicates.CREATURES
+                            .and(CardPredicates.PLANESWALKERS) //TODO: Should this be "or" Planeswalkers?
+                            .and(CardPredicates.hasKeyword(Keyword.INDESTRUCTIBLE))
+                            .and(CardPredicates.isTargetableBy(sa))
+            );
```

Comment:
Per the TODO, should this be "creatures or planeswalkers"? The way it was written previously would only find cards that have both types.

Reply:
yea no doubt OR is the better choice